src/couch_jobs/test/couch_jobs_tests.erl

```diff
@@ -426,6 +431,30 @@ resubmit_enqueues_job(#{t1 := T, j1 := J}) ->
     end).
 
 
+resubmit_finished_updates_job_data(#{t1 := T, j1 := J}) ->
```

Comment:
Let's do one more case where we resubmit but with undefined data (resubmit/3) and then data should not be updated.